dev/io.openliberty.netty.internal.impl/src/io/openliberty/netty/internal/tcp/TCPConfigurationImpl.java

```diff
@@ -58,7 +59,8 @@
     protected static final String ZAIO_RESOLVE_FOREIGN_HOSTNAMES_KEY = "zaioResolveForeignHostnames";
     protected static final String ZAIO_FREE_INITIAL_BUFFER_KEY = "zaioFreeInitialBuffers";
 
-    private static final TraceComponent tc = Tr.register(TCPConfigurationImpl.class, TCPMessageConstants.NETTY_TRACE_NAME, TCPMessageConstants.TCP_BUNDLE);
+    private static final TraceComponent tc = Tr.register(TCPConfigurationImpl.class,
+            TCPMessageConstants.NETTY_TRACE_NAME, TCPMessageConstants.TCP_BUNDLE);
```

Comment:
This seems inconsistent with all the other registration of logs throughout the PR (in the SIP side of things). Might be worth revisiting the use of Tr vs Log, and how we are registering the loggers. The approach on this line seems to take into account a Netty trace convention. Would be worth in my opinion to have classes with a set alias so that the log could be set by package name or by using this alias. Easier to say ExampleChannel=all than ibm.com.ws.example.*=all.

Reply:
It's consistent within the io.openliberty.netty.* classes.  True in the SIP classes its a mix of log classes.  Will look at something like Netty=all as a trace option.